worlds/dark_souls_3/docs/setup_en.md

```diff
@@ -21,7 +21,19 @@ This client has only been tested with the Official Steam version of the game at
 
 ## Downpatching Dark Souls III
 
-Follow instructions from the [speedsouls wiki](https://wiki.speedsouls.com/darksouls3:Downpatching) to download version 1.15. Your download command, including the correct depot and manifest ids, will be "download_depot 374320 374321 4471176929659548333"
+To downpatch DS3 for use with Archipelago, use the following instructions from the speedsouls wiki database. 
+
+1. Launch Steam (in online mode)
```

Comment:
Small nitpick, every subsequent step seem to be ending with a period but the first one is not
```suggestion
1. Launch Steam (in online mode).
```